includes/class.options.php

```diff
@@ -498,7 +498,7 @@ function get_key() {
 	function &add( $id, $properties = array(), $update = false ) {
 		// Create item
 		$args = func_get_args();
-		$ret  = call_user_func_array( array( 'parent', 'add' ), $args );
```

Comment:
I resolved this with this code: 

$ret = parent::add(...func_get_args());

I was coming to see if it was implemented.. but I guess both ways work. Cheers!

Reply:
yes, both seem to work. I don't mind updating my PR to that style.

Reply:
@archetyped do you have a preference, and are you interested in merging this PR?